README.md

```diff
@@ -1,15 +1,73 @@
-## What's this?
+## Vanilla Docker environment
 
+*Only supports Mac OSX at the moment but everything can be easily adapted to work on other platforms.*
+*Assume that you are using [PhpStorm](https://www.jetbrains.com/phpstorm/) as your IDE.*
 
+This repository contains a ready-for-development environment to develop against Vanilla.
+
+## The containers
+
+### db 
+*This container also has the alias "database".*
+
+Percona SQL database.
+
+- Accessible from the container with the hosts "db" or "database"
+- Accessible from the docker host machine with the hosts "db", "database", "localhost", "127.0.0.1'
+- The user is "root" and there is no password.
+
+### httpd
+
+Apache 2 web server.
+
+- Accessible with the port 9080 and 9443
+- Serve https://dev.vanilla.localhost:9443
+
+### nginx
+
+nginx web server
+
+- Serve:
+    - https://dev.vanilla.localhost (main forum)
+    - https://sso.vanilla.localhost (sso providers)
+    - http://vanilla.test:8080 (unit tests address)
+
+### php-fpm
+
+php-fpm with PHP 7.1
 
 ## Setup
 
-1. Get [Docker for OSX](https://download.docker.com/mac/stable/Docker.dmg).
-1. 
+*For this setup to work properly you need to clone all vanilla repositories in the same base directory*
```

Comment:
It might be a good idea to illustrate a little tree structure here like so:
```
~/workspace
|-- vanilla
|-- vanilla-docker
|-- addons
```

A note that symlinks have to be relative would be a good idea too.